### PR TITLE
Implemented Time page

### PR DESCRIPTION
**Pull request by sonalranjit**
implemented time page component
- added `react-datepicker` for date/time picking

### CODE REVIEW

File: bikespace_frontend/src/components/Time.tsx [hunk 1]
```diff
@@ -1,5 +1,79 @@
...
+                duration = ParkingDuration.Overnight;
+                break;
+        }
+        props.onParkingDurationChanged(duration);
+    };
+
+    return (
+        <div id="submission-time">
+            <h2>When did this happen?</h2>
+            <input value={dateTime.toISOString().slice(0,-8)} type="datetime-local" onChange={handleDateTime}>
```
Comment (kevinluo201) [inline bikespace_frontend/src/components/Time.tsx]: I think it'll use UTC time. I also tried to do that and I found it's ridiculously difficult!

Comment (kevinluo201) [reply bikespace_frontend/src/components/Time.tsx]: I saw people use 2 libraries, moment and moment-timezone to solve this problem. It's bad, no wonder no one really like javascript 😂

Comment (kevinluo201) [reply bikespace_frontend/src/components/Time.tsx]: image

**Review by kevinluo201 — APPROVED** [review]
@sonalranjit I solved it by composing the date time one field by one field 😂

**Comment by sonalranjit** [comment]
Hahah amazing @kevinluo201, whatever works!

**Pull request merged**